wherehows-web/.eslintrc.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
   },
   rules: {
     'no-implicit-globals': [1],
+    'typescript/class-name-casing': 'error',
+    'typescript/interface-name-prefix': ['error', 'always'],
+    'typescript/member-delimiter-style': 'error',
+    'typescript/no-empty-interface': 'error',
```

Comment:
Does empty interface prevent you from creating an empty object or does it just apply to ts interface declarations?

Reply:
just interfaces